pkg/syncer/cnsoperator/controller/cnsfileaccessconfig/cnsfileaccessconfig_controller.go

```diff
@@ -363,70 +362,68 @@ func (r *ReconcileCnsFileAccessConfig) Reconcile(ctx context.Context, request re
 	}
 	if !removePermission {
 		if len(clientVms) == 0 {
 			err = r.configureVolumeACLs(ctx, volumeID, tkgVMIP, false)
 			if err != nil {
-				msg := fmt.Sprintf("Failed to add net permissions for file volume %q. Error: %+v", volumeID, err)
-				log.Error(msg)
-				return errors.New(msg)
+				return logger.LogNewErrorf(log, "Failed to add net permissions for file volume %q. Error: %+v",
+					volumeID, err)
 			}
 		}
-		err = cnsFileVolumeClientInstance.AddClientVMToIPList(ctx, instance.Namespace+"/"+instance.Spec.PvcName, instance.Spec.VMName, tkgVMIP)
+		err = cnsFileVolumeClientInstance.AddClientVMToIPList(ctx,
+			instance.Namespace+"/"+instance.Spec.PvcName, instance.Spec.VMName, tkgVMIP)
 		if err != nil {
-			msg := fmt.Sprintf("Failed to add VM %q with IP %q to IPList. Error: %+v", vm.Name, tkgVMIP, err)
-			log.Error(msg)
-			return errors.New(msg)
+			return logger.LogNewErrorf(log, "Failed to add VM %q with IP %q to IPList. Error: %+v",
+				vm.Name, tkgVMIP, err)
 		}
-		log.Debugf("Successfully added VM IP %q to IPList for CnsFileAccessConfig request with name: %q on namespace: %q", tkgVMIP, instance.Name, instance.Namespace)
+		log.Debugf("Successfully added VM IP %q to IPList for CnsFileAccessConfig request with name: %q on namespace: %q",
+			tkgVMIP, instance.Name, instance.Namespace)
 		return nil
 	}
-	// removePermission is set to true
+	// RemovePermission is set to true.
 	if len(clientVms) == 1 && clientVms[0] == vm.Name {
 		err = r.configureVolumeACLs(ctx, volumeID, tkgVMIP, true)
 		if err != nil {
-			msg := fmt.Sprintf("Failed to remove net permissions for file volume %q. Error: %+v", volumeID, err)
-			log.Error(msg)
-			return errors.New(msg)
+			return logger.LogNewErrorf(log, "Failed to remove net permissions for file volume %q. Error: %+v",
+				volumeID, err)
 		}
 	}
-	err = cnsFileVolumeClientInstance.RemoveClientVMFromIPList(ctx, instance.Namespace+"/"+instance.Spec.PvcName, instance.Spec.VMName, tkgVMIP)
+	err = cnsFileVolumeClientInstance.RemoveClientVMFromIPList(ctx,
+		instance.Namespace+"/"+instance.Spec.PvcName, instance.Spec.VMName, tkgVMIP)
 	if err != nil {
-		msg := fmt.Sprintf("Failed to remove VM %q with IP %q to IPList. Error: %+v", vm.Name, tkgVMIP, err)
-		log.Error(msg)
-		return errors.New(msg)
+		return logger.LogNewErrorf(log, "Failed to remove VM %q with IP %q to IPList. Error: %+v", vm.Name, tkgVMIP, err)
 	}
-	log.Debugf("Successfully removed VM IP %q to IPList for CnsFileAccessConfig request with name: %q on namespace: %q", tkgVMIP, instance.Name, instance.Namespace)
+	log.Debugf("Successfully removed VM IP %q to IPList for CnsFileAccessConfig request with name: %q on namespace: %q",
+		tkgVMIP, instance.Name, instance.Namespace)
 	return nil
 }
 
 // configureVolumeACLs helps to prepare the CnsVolumeACLConfigureSpec
 // for a given TKG VM IP address and volumeID and invoke CNS API
```

Comment:
Should there be a full stop here?

Reply:
The major purpose of this change is LogNewErrorCodef. I only do cleanup opportunistically in this change.

Since you noticed this, I would fix it in the same change. Done.